config/jobs/cert-manager/cert-manager/release-previous/cert-manager-release-previous-presubmits.yaml

```diff
@@ -927,7 +921,6 @@ presubmits:
 
 # Run with Bazel for release-1.7 where make was not available yet
   - name: pull-cert-manager-e2e-v1-24
-    context: pull-cert-manager-make-e2e-v1-24
```

Comment:
This is an example copy paste error - job's name in Github UI wil be `pull-cert-manager-make-e2e-v1-24`